scss/_patterns_switch.scss

```
@@ -31,25 +35,27 @@ $knob-size: $sp-unit * 3;
   .p-switch__slider {
     @extend %vf-has-round-corners;
 
-    background: linear-gradient(to right, $color-link 50%, $color-mid-light 50%);
     box-shadow: inset 0 2px 5px 0 transparentize($color-dark, 0.8);
     display: inline-block;
     height: $knob-size;
     margin: 0;
     position: relative;
     width: $knob-size * 2;
+    border-radius: .7rem;
```

```suggestion
    border-radius: math.div($knob-size, 2);
```

And add `@use sass:math;`